Library/Homebrew/cmd/audit.rb

```diff
@@ -234,6 +234,32 @@ def audit_options
     end
   end
 
+  def audit_desc
+    # For now, only check the description when using `--strict`
+    return unless @strict
+
+    desc = formula.desc
+
+    unless desc
+      problem "All formulas should have a desc."
+      return
+    end
+
+    # Make sure the formula name plus description is no longer than 80 characters
+    linelength = formula.name.length + ": ".length + desc.length
+    if linelength > 80
+      problem "Description is too long. \"name: desc\" should be less than 80 characters (currently #{linelength})."
+    end
+
+    if desc =~ %r[[Cc]ommandline]
+      problem "It should be spelled \"command-line\", not \"commandline\"."
```

Comment:
To avoid getting into the British/American syntax discussion, we could probably drop the `spelled` from this and the line below entirely. The example given is explicit enough not to need it.